users/templates/account/login.html

```diff
@@ -2,8 +2,15 @@
 
 {% load static bootstrap4 %}
 
+{% block extrastyles %}
+<link href="{% static 'login.css' %}" rel="stylesheet">
+{% endblock %}
+
 {% block content %}
   <form class="form-horizontal" role="form" action="{% url 'account_login' %}" method="POST">
+    {% if form.non_field_errors %}
```

Comment:
I think you should put this alert between `Sign in` header and divider or even below divider.